### PR TITLE
[PLA-95] Mobile fixes/improvements

### PR DESCRIPTION
**Pull request by MythicalFish**

### CODE REVIEW

**Comment by MythicalFish** [comment]
@hanchang All done :) The coins index is still pretty cramped though. Should I maybe hide an additional column?

Side note: I think that the design/UI could be improved a lot more (both on desktop & mobile), especially the watchlist page, but I'd need to spend some time trying things out and coming up with designs.

**Comment by hanchang** [comment]
@MythicalFish 💯 LGTM! Feel free to merge and deploy at will :)

File: app/helpers/coins_helper.rb [hunk 1]
```diff
@@ -18,4 +19,24 @@ def custom_number_to_currency(amount, options = {})
   def display_available_supply(coin)
     coin.available_supply.try(:>, 0) ? "#{number_with_delimiter(coin.available_supply)} #{coin.symbol}" : ""
   end
+
+  def humanize input, prefix = '', suffix = ''
```
Comment (hanchang) [inline app/helpers/coins_helper.rb]: Thanks for the refactor! ✨

File: app/views/coins/icos.slim [hunk 2]
```diff
@@ -1,42 +1,73 @@
...
-              td.tr= coin.max_supply
-              td.tr
-                - if coin.ico_token_price_usd
-                  => number_to_currency(coin.ico_token_price_usd, precision: 4)
-                  small USD
-                - else
-                  | ?
-              td.tr= coin.token_type
+                  label
+                    | Hard cap
```
Comment (hanchang) [inline app/views/coins/icos.slim]: @MythicalFish Actually can we capitalize this? "Hard Cap"

**Pull request merged**